src/machines/java/com/enderio/machines/common/blockentity/base/MachineBlockEntity.java

```diff
@@ -238,7 +242,7 @@ public boolean canAct() {
 
     public boolean canActSlow() {
         return canAct()
-            && level.getGameTime() % 5 == 0;
+                && level.getGameTime() % 5 == 0;
```

Comment:
This is because I am coding with VSCode, and the `.editorconfig` values don't seem to be being respected. I did download IntelliJ to check, and it put this on one line, which is different from what this was originally. I have just left it as one line.